rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
```

Comment:
- [ ] Kindly make `book` and `person` attributes have getter and setter methods. 👍

Reply:
Why do we need a setter method for `book` and `person` here? I don't think a setter method for the `date` is required here. Anyways I followed the example here. 

```
class Visit
  attr_reader :animal, :vet
  attr_accessor :date

  def initialize(date, animal, vet)
    @date = date

    @animal = animal
    animal.visits << self

    @vet = vet
    vet.visits << self
  end
end
```

Reply:
Hi @awaisanwar544 👋 , You need a setter and getter method for the `Date`, `Person` and `Book`. Like I said earlier, if you are not satisfied with the review, you can kindly request a second opinion and a senior code reviewer will have a look at it. 👍